Cargo.toml

```diff
@@ -1,28 +1,36 @@
 [workspace]
 resolver = "2"
-members = ["demo"]
+members = ["examples/with_winit", "examples/run_wasm", "examples/scenes"]
 
 [workspace.package]
 edition = "2021"
-version = "0.0.1"
-license = "MIT OR Apache-2.0"
+version = "0.1.0"
+license = "Apache-2.0 OR MIT"
 repository = "https://github.com/linebender/velato"
 
```

Comment:
```suggestion
publish = false
```
This is likely to be publishable as-is, but it's better to make sure we have consensus before doing that (e.g. an opportunity to review the final state of the readme, etc.)

Reply:
This doesn't stop someone from publishing, e.g. "cargo publish --allow-dirty" - I'd rather just have a conversation before publishing but I'll keep the comments on this open.

Reply:
This suggestion is not about stopping someone malicious publishing - we can't do that

It's specifically about making sure there's consensus for an initial release. E.g. I'd want Chad to sign off on making a release, whereas this PR doesn't require their input.

Reply:
@dfrg

Reply:
Whether that consensus is gained on GitHub or on Zulip, it's easier to track and record that on GitHub with the standardised approval button

Reply:
I’m mostly ambivalent on this but adding a tiny safeguard makes sense to me so +1 to Daniel’s suggestion.